src/components/ux-radio/ux-radio.hbs

```diff
@@ -0,0 +1,12 @@
+
+    <label>{{.label}}</label>
+    {{#each items}}
+        <input {{#if .type}}type="{{.type}}"{{/if}}
+            {{#if .name}}name="{{.name}}"{{/if}}
+            {{#if .id}}id="{{.id}}"{{/if}}
+            {{#if .value}}value="{{.value}}"{{/if}}
+               on-blur="handleBlur"
+               on-tap="handleTap">
```

Comment:
If these are validation purposes you can merge them like `on-blur-tap="validate"` or something similar.